### PR TITLE
Add ability to configure UserAgent

### LINKED ISSUE #175: Is there any way to change User-Agent?
As I see such option present on the CefGlue (by `Cef.Initialize(mySettingsWithUserAgent);`) but in your wrapper implementation InitializeCef is private and I can't change User-Agent in any way

### PR DESCRIPTION
**Pull request by Takoooooo**
I think the changes I made are obvious so I don't think I need to add any description. Also if needed I may revert all changes made in WebView.Control.Avalonia.csproj,i made them because I cant build it in any other way.Also, I'm not sure what is better, to add all properties from CefSettings to WebView class or just to add method which will have CefSettings as argument, just made it in the way it was previously done

Will close #175

### CODE REVIEW

File: WebViewControl.Avalonia/WebViewControl.Avalonia.csproj [hunk 1]
```diff
@@ -67,8 +67,8 @@
 
   <ItemGroup>
     <!--When we upload Avalonia here, we should increase the version of both WebViewControl.Avalonia as well as ReactViewControl.Avalonia-->  
-    <PackageReference Include="Avalonia" Version="0.10.0-rc1" />  
-    <PackageReference PrivateAssets="None" Include="CefGlue.Avalonia" Version="85.4183.9" />
+    <PackageReference Include="Avalonia" Version="0.10.0-preview6" />
```
Comment (joaompneves) [inline WebViewControl.Avalonia/WebViewControl.Avalonia.csproj:70]: please revert these changes, otherwise I cant merge
already published the new cefglue packages on nuget

File: WebViewControl/WebView.cs [hunk 2]
```diff
@@ -99,7 +99,7 @@ public partial class WebView : IDisposable {
         private string CurrentDomainId { get; }
 
         private string DefaultLocalUrl { get; }
-
+        public static string UserAgent { get; set; }
```
Comment (joaompneves) [inline WebViewControl/WebView.cs:102]: missing spacing before and after

File: WebViewControl/WebView.cs [hunk 3]
```diff
@@ -119,13 +119,15 @@ public partial class WebView : IDisposable {
...
-            cefSettings.WindowlessRenderingEnabled = OsrEnabled;
-            cefSettings.RemoteDebuggingPort = GetRemoteDebuggingPort();
+            var cefSettings = new CefSettings {
+                LogSeverity = string.IsNullOrWhiteSpace(LogFile) ? CefLogSeverity.Disable : (EnableErrorLogOnly ? CefLogSeverity.Error : CefLogSeverity.Verbose),
+                LogFile = LogFile,
+                UncaughtExceptionStackSize = 100, // enable stack capture
+                CachePath = CachePath, // enable cache for external resources to speedup loading
+                WindowlessRenderingEnabled = OsrEnabled,
+                RemoteDebuggingPort = GetRemoteDebuggingPort(),
+                UserAgent= UserAgent
```
Comment (joaompneves) [inline WebViewControl/WebView.cs:129]: missing spacing before =

**Review by joaompneves — CHANGES_REQUESTED** [review]

**Comment by Takoooooo** [comment]
now it should be ok i think

**Review by joaompneves — APPROVED** [review]
